src/components/LoginDropDownComponent/DropDownComponent.js

```diff
@@ -63,41 +64,45 @@ function DropdownComponent() {
         if (storedData) {
             setUserDetails(JSON.parse(storedData));
             setIsLoggedIn(true);
+        } else {
+            setIsLoggedIn(false);
+            setUserDetails(null); // Reset userDetails
         }
-    }, [isLoggedIn]);
+    }, [isLoggedIn]); // Now useEffect depends on isLoggedIn
 
     function logout() {
+        console.log('Logging out...');
```

Comment:
Is this required for prod?

Reply:
by mistake i pushed tihs remove in the new version